api/config/cdm/cdm_5_2.json

```diff
@@ -934,6 +934,25 @@
           "name": "qualifier_source_value",
           "mode": "nullable",
           "description": "The source value associated with a qualifier to characterize the observation"
+        },
+        {
+          "type": "integer",
+          "name": "value_source_concept_id",
+          "mode": "nullable",
+          "description": "A foreign key to a Concept for the value in the source data. This is applicable to observations where the result can be expressed as a non-standard concept.",
+          "foreignKey": "concept"
```

Comment:
Could you please add something to the README which describes where these came from? Were these not in the JSON file?

Reply:
Done.